packages/utils/src/db/db.ts

```diff
@@ -708,33 +712,103 @@ export class HoprDB {
   }
 
   /**
-   * Store entry if account is elegible, if not delete entry.
+   * Hopr Network Registry
+   * Link hoprNode to an ETH address.
+   * @param hoprNode the node to register
+   * @param account the account that made the transaction
+   * @param snapshot
+   */
+  public async addToRegistry(hoprNode: PublicKey, account: Address, snapshot: Snapshot): Promise<void> {
+    await this.db
+      .batch()
+      .put(Buffer.from(this.keyOf(createWhitelistRegistryKey(hoprNode))), Buffer.from(account.serialize()))
+      .put(Buffer.from(LATEST_CONFIRMED_SNAPSHOT_KEY), Buffer.from(snapshot.serialize()))
+      .write()
+  }
+
+  /**
+   * Hopr Network Registry
+   * Unlink hoprNode to an ETH address by removing the entry.
+   * @param hoprNode the node to register
+   * @param snapshot
+   */
+  public async removeFromRegistry(account: Address, snapshot: Snapshot): Promise<void> {
+    // range of keys to search
+    const from = this.keyOf(
+      Uint8Array.from([...WHITELIST_REGISTRY_PREFIX, ...new Uint8Array(PublicKey.SIZE_COMPRESSED).fill(0x00)])
```

Comment:
```suggestion
      Uint8Array.from([...WHITELIST_REGISTRY_PREFIX, ...new Uint8Array(PublicKey.SIZE_UNCOMPRESSED).fill(0x00)])
```

Reply:
Probably not necessary due to https://github.com/hoprnet/hoprnet/pull/3704#discussion_r838643200